dev/src/overmind/actions.tsx

```diff
@@ -616,7 +650,10 @@ export const isAuthorizedTeacher = async ({ effects }: Context): Promise<boolean
 }
 
 export const alertHandler = ({ state }: Context, response: IGrpcResponse<unknown>): void => {
-    if (response.status.getCode() >= 0) {
+    if (response.status.getCode() === StatusCode.UNAUTHENTICATED) {
+        // The user is not logged in.
+        state.alerts.push({ text: "Your session is not valid. Please refresh this page and log in to continue.", color: Color.RED })
```

Comment:
Does this mean were logged in, but the session became invalid... If so, then
```text
Your session has expired. Please reload page to log in.
```
On the other hand, if we get here, would it not be possible to trigger a reload of the page? I guess then we should instead just say:
```text
Your session has expired. Please log in again.
```